### ISSUE TITLE
Since moving server, Discourse data source is no longer importing?

### ISSUE THREAD

**Issue by RCheesley** [chaoss/grimoirelab-sirmordred#374]
Prior to moving, data was flowing in perfectly to Discourse, but now nothing is coming in.

Github and Slack are flowing just fine, but the data source does not show any activity for Discourse.

I'm a bit confused as to what might be the problem given all the other data sources are working fine! Any ideas?

![screenshot-157 245 245 139_5601-2019 11 16-16_24_21](https://user-images.githubusercontent.com/2930593/68996115-a29f9600-088d-11ea-8331-40e32a75b415.png)

**Comment by valeriocos** [comment]
Hi @RCheesley, can you share your projects.json and the other config files?

If possible, could you also paste here the output of the following urls:
- your-elasticsearch-url/_cat/indices?pretty
- your-elasticsearch-url/_alias?pretty

Thanks!

**Comment by RCheesley** [comment]
Hi there,

Projects file: https://pastebin.com/F1up4T2C
setup.cfg: https://pastebin.com/rMSpguBy

First output:
```
yellow open github_demo_enriched    t7T1a7EaRA6s4jPzXmInyw 5 1   9332     0  13.1mb  13.1mb
yellow open git_demo_enriched       cMh0M3SFS6y-T6T4Mht_xQ 5 1  24417 13006  55.7mb  55.7mb
yellow open git_demo_raw            8-m1JJAATuWK4S0MEGMFIw 5 1  24417     0  43.9mb  43.9mb
yellow open github_demo_raw         obfTbmSlTnSRNzWkjmvJ_w 5 1   9332    28  72.4mb  72.4mb
yellow open discourse_demo_raw      eHHFknASTKai-jl12-5LSg 5 1   7059    15  42.9mb  42.9mb
yellow open slack_demo_enriched     gpxptrXaREy276-PKRv8Mg 5 1 118973 25980 122.6mb 122.6mb
yellow open .kibana                 hgiSdf9iRFOUtJ9FcuyVoA 1 1    205    37 316.6kb 316.6kb
yellow open git-onion_demo_enriched ZZUE3Ug6QWC0oaC75OQeGA 5 1   5006     0   1.4mb   1.4mb
yellow open git-aoc_demo_enriched   BeqKY8y4QQe4XncZ6egjyA 5 1 375872 96274 280.5mb 280.5mb
yellow open discourse_demo_enriched 1f5unxpASoWQjY5YyiMHDA 5 1      0     0   1.2kb   1.2kb
yellow open slack_demo_raw          1WZxVPuCR2KZd-33PQArVQ 5 1 118973    80   412mb   412mb
```

Second output:
```
{
  "github_demo_enriched" : {
    "aliases" : {
      "github_issues" : { },
      "github_issues_enrich" : { },
      "issues_closed" : { },
      "issues_created" : { },
      "issues_updated" : { }
    }
  },
  "git_demo_enriched" : {
    "aliases" : {
      "affiliations" : { },
      "demographics" : { },
      "git" : { },
      "git_author" : { },
      "git_enrich" : { }
    }
  },
  "git_demo_raw" : {
    "aliases" : {
      "git-raw" : { }
    }
  },
  "github_demo_raw" : {
    "aliases" : {
      "github-raw" : { }
    }
  },
  "discourse_demo_raw" : {
    "aliases" : {
      "discourse-raw" : { }
    }
  },
  ".kibana" : {
    "aliases" : { }
  },
  "git-onion_demo_enriched" : {
    "aliases" : {
      "all_onion" : { }
    }
  },
  "slack_demo_enriched" : {
    "aliases" : {
      "slack" : { }
    }
  },
  "git-aoc_demo_enriched" : {
    "aliases" : {
      "git_areas_of_code" : { }
    }
  },
  "discourse_demo_enriched" : {
    "aliases" : {
      "discourse" : { }
    }
  },
  "slack_demo_raw" : {
    "aliases" : {
      "slack-raw" : { }
    }
  }
}
```

**Comment by valeriocos** [comment]
Thank you @RCheesley , I'll have a look at it ASAP

**Comment by valeriocos** [comment]
I checked the conf files and they look OK. I'm now executing the collection and enrichment phases for discourse, maybe there is an error in the enrichment phase.

Which version of grimoirelab are you using?

**Comment by RCheesley** [comment]
Here's the docker-compose file: https://pastebin.com/7jCf6eDj

**Comment by valeriocos** [comment]
Great, maybe we found the issue! the docker-compose is using the version 0.2.31 of grimoirelab (so you are bumping into this bug https://github.com/chaoss/grimoirelab-elk/issues/701 that was fixed in 0.2.32).

Please update the line grimoirelab image to `image: bitergia/mordred:grimoirelab-0.2.32`.

To follow up with https://github.com/chaoss/grimoirelab-sirmordred/issues/374#issuecomment-557056436, I executed it locally and no error popped up.

**Comment by RCheesley** [comment]
OK, will give it a go.

**Comment by RCheesley** [comment]
Maybe someone might want to update https://gitlab.com/Bitergia/lab/analytics-demo/ :)

**Comment by valeriocos** [comment]
good catch @RCheesley , maybe we can use the tag `latest`

**Comment by sanacl** [comment]
> good catch @RCheesley , maybe we can use the tag `latest` (cc @sanacl )

Yep, we should!

**Comment by RCheesley** [comment]
> Great, maybe we found the issue! the docker-compose is using the version 0.2.31 of grimoirelab (so you are bumping into this bug [chaoss/grimoirelab-elk#701](https://github.com/chaoss/grimoirelab-elk/issues/701) that was fixed in 0.2.32).
> 
> Please update the line grimoirelab image to `image: bitergia/mordred:grimoirelab-0.2.32`.
> 
> To follow up with [#374 (comment)](https://github.com/chaoss/grimoirelab-sirmordred/issues/374#issuecomment-557056436), I executed it locally and no error popped up.

This fixed the issue, Discourse is now being brought into data sources and correctly enriched, hurrah!
